src/rust/engine/sharded_lmdb/src/lib.rs

```diff
@@ -113,15 +116,15 @@ impl ShardedLmdb {
       fs::safe_create_dir_all(&dir)
         .map_err(|err| format!("Error making directory for store at {:?}: {:?}", dir, err))?;
       envs.push((
-        ShardedLmdb::make_env(&dir, max_size)?,
+        ShardedLmdb::make_env(&dir, max_size_bytes)?,
         dir,
         fingerprint_prefix,
       ));
     }
     Ok(envs)
   }
 
-  fn make_env(dir: &Path, max_size: usize) -> Result<Environment, String> {
+  fn make_env(dir: &Path, max_size_bytes: usize) -> Result<Environment, String> {
```

Comment:
`page_size` seems like it would be relevant when doing perf tuning and benchmarking, could we add a log statement that logged the value as a debug when we create an lmdb? Something like:

```suggestion
  fn make_env(dir: &Path, max_size_bytes: usize) -> Result<Environment, String> {
      debug!("Creating environment with page size {}", page_size::get());
```

(assuming calls to page_size are both idempotent and cheap)

Reply:
I'll hold off on adding this until we want the data for something specific :)